server/routerlicious/packages/routerlicious-base/src/alfred/app.ts

```diff
@@ -97,6 +99,24 @@ export function create(
 						[BaseTelemetryProperties.tenantId]: getTenantIdFromRequest(req.params),
 						[BaseTelemetryProperties.documentId]: getIdFromRequest(req.params),
 					};
+					if (enableClientIPLogging === true) {
```

Comment:
It would be better if this code were in a shared space, like the `jsonMorganLoggerMiddleware` method. That way Historian could also utilize it.

Reply:
Actually, this is temporary and is expected to be removed after collecting the data mentioned in the description. So, I tried to keep changes to a minimum.